lib/core/src/logger.rs

```diff
@@ -34,40 +32,48 @@ impl log::Log for GlobalSdkLogger {
 }
 
 pub(super) fn init_logging(log_dir: &str, app_logger: Option<Box<dyn log::Log>>) -> Result<()> {
-    let target_log_file = Box::new(
```

Comment:
As far as I understand this method was only created to easy the usage for rust callers. It is not exposed to the bindings and we only use it from the cli.
Perhaps then it make sense to exclude it completely for wasm?
If we think later we need something specific for wasm perhaps we can add it in the wasm crate?

Reply:
Thanks. I didn't have in mind that it's only used on the rust interface. I completely excluded it for wasm in [101751f](https://github.com/breez/breez-sdk-liquid/pull/792/commits/101751fbd4eb19702ac68fa3426419ccf547565e)